packages/twenty-front/src/modules/dashboards/components/graphs/GraphWidgetNumberChart.tsx

```diff
@@ -0,0 +1,66 @@
+import { useTheme } from '@emotion/react';
```

Comment:
folder structure should be dashboards/graphs/components (components, hooks, etc... are always leaf folders)